esti/catalog_export_test.go

```diff
@@ -509,3 +527,89 @@ func TestDeltaCatalogExport(t *testing.T) {
 	require.Equal(t, "delta_exporter", tasks.JSON200.Results[0].HookId)
 	validateExportTestByStorageType(t, ctx, headCommit.Id, testData, blockstore)
 }
+
+func TestDeltaCatalogImportExport(t *testing.T) {
+	ctx, _, repo := setupTest(t)
+	defer tearDownTest(repo)
+
+	requireBlockstoreType(t, block.BlockstoreTypeS3)
+	accessKeyID := viper.GetString("access_key_id")
+	secretAccessKey := viper.GetString("secret_access_key")
```

Comment:
Can we use testutil.SetupTestingEnv here?

Reply:
In what sense?
testutil.SetupTestingEnv configures the values (it is used in the TestMain and is run on every test)
Here we are just retrieving the values after running testutil.SetupTestingEnv